### PR TITLE
Riedl

### PR DESCRIPTION
**Pull request by jriedl**

### CODE REVIEW

File: cpp/zotaut/sync_qs_settings.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Create a new temporary database to work on, create an id mapping an create a new table with correct original journal ids
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) --execute "CREATE DATABASE ${TMP_DATABASE}"
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" < $FILE_TO_SYNC
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute  "CREATE TABLE ${TMP_DATABASE}.id_mapping SELECT ${DATABASE}.zeder_journals.id AS orig, ${TMP_DATABASE}.zeder_journals.id AS new FROM ${TMP_DATABASE}.zeder_journals JOIN ${DATABASE}.zeder_journals ON ${TMP_DATABASE}.zeder_journals.zeder_instance=${DATABASE}.zeder_journals.zeder_instance AND ${TMP_DATABASE}.zeder_journals.zeder_id=${DATABASE}.zeder_journals.zeder_id COLLATE utf8mb4_general_ci"
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute "CREATE TABLE ${TMP_DATABASE}.metadata_presence_tracer_new SELECT ${TMP_DATABASE}.id_mapping.orig AS journal_id, marc_field_tag, marc_subfield_code, regex, record_type, field_presence FROM id_mapping RIGHT JOIN metadata_presence_tracer ON id_mapping.new=metadata_presence_tracer.journal_id"
```
Comment (mtrojan-ub) [inline cpp/zotaut/sync_qs_settings.sh:31]: In this line, a new table `${TMP_DATABASE}.metadata_presence_tracer_new` is created, which seems neither being used nor cleaned-up later

File: cpp/zotaut/sync_qs_settings.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Backup
+
```
Comment (mtrojan-ub) [inline cpp/zotaut/sync_qs_settings.sh:34]: superfluous empty line

File: cpp/zotaut/sync_qs_settings.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute "DROP TABLE IF EXISTS ${DATABASE}.metadata_presence_tracer_${CURRENT_DATE}"
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute "CREATE TABLE ${DATABASE}.metadata_presence_tracer_${CURRENT_DATE} SELECT * FROM metadata_presence_tracer"
+
+#Replace the original
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute "DROP TABLE ${DATABASE}.metadata_presence_tracer"
+mysql --defaults-extra-file=<(echo ${MYSQL_LOGIN_PARAMS}) "${TMP_DATABASE}" --execute "CREATE TABLE ${DATABASE}.metadata_presence_tracer SELECT * FROM ${TMP_DATABASE}.metadata_presence_tracer"
```
Comment (mtrojan-ub) [inline cpp/zotaut/sync_qs_settings.sh:40]: should `${TMP_DATABASE}.metadata_presence_tracer_new` be used here instead?

File: cpp/zotaut/sync_qs_settings.sh (hunk 1 above)
Comment (jriedl) [reply cpp/zotaut/sync_qs_settings.sh:31]: see below

**Pull request merged**